src/pages/Dashboard.tsx

```diff
@@ -33,99 +24,149 @@ function Dashboard() {
-              margin="dense"
-              id="name"
-              name="name"
-              label="Name"
-              type="text"
-              fullWidth
-              variant="standard"
-            />
-            <TextField
-              autoFocus
-              margin="dense"
-              id="description"
-              name="description"
-              label="Description"
-              type="text"
-              fullWidth
-              variant="standard"
-            />
-          </DialogContent>
-          <DialogActions>
-            <Button onClick={handleClose}>Cancel</Button>
-            <Button type="submit">Create</Button>
-          </DialogActions>
-        </Dialog>
-      </div>
+
+        {open && (
+          <div
+            role="dialog"
+            aria-modal="true"
+            className="modal-overlay"
+            onClick={handleClose}
+          >
+            <div className="modal-content" onClick={(e) => e.stopPropagation()}>
+              <h2 className="modal-title">Create new project</h2>
+              <form
+                onSubmit={(event: React.FormEvent<HTMLFormElement>) => {
+                  event.preventDefault();
+                  const formData = new FormData(event.currentTarget);
+                  const formObject = Object.fromEntries(formData.entries());
+                  const data: AddProjectData = {
+                    name: String(formObject.name || '').trim(),
+                    description: String(formObject.description || '').trim(),
+                  };
+                  if (!data.name) return;
+                  addProject.mutate(data, {
+                    onSuccess: () => {
+                      setOpen(false);
+                    },
+                  });
+                }}
+              >
+                <div className="form-row">
+                  <label htmlFor="name" className="form-label">
+                    Name
+                  </label>
+                  <input
+                    id="name"
+                    name="name"
+                    required
+                    autoFocus
+                    type="text"
+                    className="input"
+                  />
+                </div>
+                <div className="form-row">
+                  <label htmlFor="description" className="form-label">
+                    Description
+                  </label>
+                  <input
+                    id="description"
+                    name="description"
+                    type="text"
+                    className="input"
+                  />
+                </div>
+
+                <div className="modal-actions">
+                  <button
+                    type="button"
+                    onClick={handleClose}
+                    disabled={addProject.status === 'pending'}
+                    className="btn btn-secondary"
+                  >
+                    Cancel
+                  </button>
```

Comment:
tutaj chyba brakuje stylow + chcemy zeby to sie tak otwieralo na dole? bardziej myslalem zeby to byl taki popup/dialog jak bylo wczesniej

<img width="455" height="258" alt="Image" src="https://github.com/user-attachments/assets/fc2c385c-58c9-4bb0-9835-ffd9d965c594" />